glean-core/ios/sdk_generator.sh

```diff
@@ -147,7 +153,7 @@ VENVDIR="${SOURCE_ROOT}/.venv"
 # We need at least pip 20.3 for Big Sur support, see https://pip.pypa.io/en/stable/news/#id48
 # Latest pip is 21.0.1
 "${VENVDIR}"/bin/pip install "pip>=20.3"
-"${VENVDIR}"/bin/pip install --upgrade glean_parser==$GLEAN_PARSER_VERSION
+"${VENVDIR}"/bin/pip install --upgrade "git+ssh://git@github.com/badboy/glean_parser@build-date#egg=glean-parser"
```

Comment:
yes, as said I want a code review here before I land the corresponding code, because if anything changes it might also require glean_parser changes ,etc.

Reply:
Got it, makes sense